src/Compilers/Core/Portable/EncodedStringText.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.CodeAnalysis.Text
 {
-    internal static class EncodedStringText
+    public static class EncodedStringText
```

Comment:
Or perhaps just an overload for From with another bool parameter "ansiDefaultEncoding" or something like that.